classes/ActionScheduler_Store.php

```diff
@@ -120,12 +127,37 @@
 	 */
 	abstract public function get_status( $action_id );
 
+	/**
+	 * @return mixed
+	 */
+	abstract public function get_claim_id( $action_id );
```

Comment:
This needs the parameter included in the docblock. Also, should this be a defined type, such as an int?